client/utils/reduxFormUtils.js

```diff
@@ -79,3 +79,14 @@ export function validateSignup(formProps) {
 
   return errors;
 }
+export function validateResetPassword(formProps) {
+  const errors = {};
+  if (!formProps.email) {
+    errors.email = 'Please enter an email.';
+  } else if (
+    // eslint-disable-next-line max-len
+    !formProps.email.match(/^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/i)) {
```

Comment:
rather than copying and pasting this regex twice, could you put it in a constant variable?